src/ipc/handlers/github_handlers.ts

```diff
@@ -627,6 +627,109 @@ async function handleDisconnectGithubRepo(
     })
     .where(eq(apps.id, appId));
 }
+// --- GitHub Clone Repo from URL Handler ---
+async function handleCloneRepoFromUrl(
+  event: IpcMainInvokeEvent,
+  {
+    url,
+    installCommand,
+    startCommand,
+  }: {
+    url: string;
+    installCommand?: string;
+    startCommand?: string;
+  },
+): Promise<{
+  success: boolean;
```

Comment:
i'd probably avoid doing this `success: boolean` for future code - i know some of the code does this pattern already, but i've found it really easy to forget to handle the failure case and i think throwing an error/exception directly is more idiomatic and easier to handle.